lib/internal/Magento/Framework/Autoload/AutoloaderInterface.php

```diff
@@ -18,7 +22,7 @@ interface AutoloaderInterface
      * @param bool $prepend Whether to append the given path or paths to the paths already associated with the prefix
      * @return void
      */
-    public function addPsr4($nsPrefix, $paths, $prepend = false);
+    public function addPsr4($nsPrefix, $paths, $prepend = false): void;
```

Comment:
@mrtuvn it's not a good idea to specify the returning type, as it will fail all the implementations. Let's only add `declare(strict_types=1)`, as you've already did, and should be enough for now.

Reply:
so I don't need add return type after every method right ? Only add strict at first line

Reply:
> so I don't need add return type after every method right ? Only add strict at first line

Exactly.

Also, looks like linking PR (Related PR) didn't work. Could you please double check that you use exactly the same way? The placeholder should also be there.

![image](https://user-images.githubusercontent.com/15868188/108813929-a1920400-75ba-11eb-8360-5432bf03c0c4.png)